.circleci/config.yml

```diff
@@ -85,6 +93,13 @@ jobs:
             PLATFORM_ENV: production
             K8S_NAMESPACE: hmcts-complaints-formbuilder-adapter-production
           command: './deploy-scripts/bin/deploy'
+      - run:
```

Comment:
we do not want to deploy to production yet

Reply:
oh yes, my bad. Have fixed this now